server/views/pages/index.njk

```diff
@@ -3,72 +3,61 @@
 {% from "govuk/components/table/macro.njk" import govukTable %}
 {% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
 
+{%- set dayjs = dayjs -%}
+{% set cases = unallocatedCases | length  %}
 {% set pageTitle = applicationName + " - Home" %}
+{% set tableData = {
+    caption: 'Unallocated community cases',
+    captionClasses: 'govuk-heading-l',
+    head: [
+        { text: 'Name / CRN' },
+        { text: 'Tier' },
+        { text: 'Sentence date' },
+        { text: 'Initial appointment' },
+        { text: 'Status' },
+        { text: 'Action' }
+    ],
+    rows: []
+} %}
+
+{% for item in unallocatedCases %}
+    {%- set tableRow = [
+        { text: item.name },
+        { text: item.tier },
+        { text: dayjs(item.sentenceDate).format('D MMM YYYY') },
```

Comment:
are we likely to format all dates the exact same throughout allocations? Could we move this to a global nunjucks function and just pass in the date?

Reply:
Dates are going to be in this format in most parts of the app so good idea, I'll update 👍